src/features/base-layout/components/header/header.test.tsx

```diff
@@ -1,22 +1,27 @@
 import { act } from 'react-dom/test-utils';
```

Comment:
```suggestion
import { act } from 'react';
```

Reply:
Corrected in [refactor: 256 - change path to act in tests](https://github.com/rolling-scopes/site/pull/258/commits/adb6fc60145cf5bc5e687ff66c08570b0193fe6d)